ui/app/templates/jobs/job/task-group.hbs

```diff
@@ -293,12 +293,17 @@
             <tr data-test-volume>
               <td data-test-volume-name>
                 {{#if row.model.isCSI}}
-                  <LinkTo
-                    @route="csi.volumes.volume"
-                    @model={{concat row.model.source "@" row.model.namespace.id}}
-                  >
-                    {{row.model.name}}
-                  </LinkTo>
+                  {{!-- if volume is per_alloc=true, there's no one specific volume. So, link to the volumes index with an active query --}}
```

Comment:
Good catch. This seems like the best we can do here.

Reply:
Probably unrelated to this PR, but on one attempt to click this link the back button broke and I couldn't get back to this page. Unfortunately it only happened once and I can't reproduce it anymore.